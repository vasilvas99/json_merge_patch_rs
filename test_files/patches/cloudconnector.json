{
    "name": "cloudconnector",
    "image": {
        "name": "ghcr.io/eclipse-leda/leda-contrib-cloud-connector/cloudconnector:main-47c01227a620a3dbd85b66e177205c06c0f7a52e"
    },
    "mounts": [
        {
            "source": "/data/var/certificates/device.crt",
            "destination": "/device.crt",
            "propagation_mode": "rprivate"
        },
        {
            "source": "/data/var/certificates/device.key",
            "destination": "/device.key",
            "propagation_mode": "rprivate"
        }
    ],
    "host_config": {
        "extra_hosts": [
            "mosquitto:host_ip"
        ]
    },
    "config": {
        "env": [
           "CERT_FILE=/device.crt",
           "KEY_FILE=/device.key",
           "LOCAL_ADDRESS=tcp://mosquitto:1883",
           "LOG_FILE=",
           "LOG_LEVEL=INFO",
           "CA_CERT_PATH=/app/iothub.crt",
           "MESSAGE_MAPPER_CONFIG=/app/message-mapper-config.json",
           "ALLOWED_LOCAL_TOPICS_LIST=cloudConnector/#"
        ]
    }
}