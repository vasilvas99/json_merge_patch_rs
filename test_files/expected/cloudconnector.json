{
    "id": "",
    "name": "cloudconnector",
    "image": {
        "name": "ghcr.io/eclipse-leda/leda-contrib-cloud-connector/cloudconnector:main-47c01227a620a3dbd85b66e177205c06c0f7a52e",
        "decrypt_config": null
    },
    "host_name": "",
    "domain_name": "",
    "resolv_conf_path": "",
    "hosts_path": "",
    "hostname_path": "",
    "mounts": [
        {
            "source": "/data/var/certificates/device.crt",
            "destination": "/device.crt",
            "propagation_mode": "rprivate"
        },
        {
            "source": "/data/var/certificates/device.key",
            "destination": "/device.key",
            "propagation_mode": "rprivate"
        }
    ],
    "hooks": [],
    "host_config": {
        "devices": [],
        "network_mode": "bridge",
        "privileged": false,
        "restart_policy": {
            "maximum_retry_count": 0,
            "retry_timeout": 0,
            "type": "unless-stopped"
        },
        "runtime": "io.containerd.runc.v2",
        "extra_hosts": [
            "mosquitto:host_ip"
        ],
        "port_mappings": [],
        "log_config": {
            "driver_config": {
                "type": "json-file",
                "max_files": 2,
                "max_size": "1M",
                "root_dir": ""
            },
            "mode_config": {
                "mode": "blocking",
                "max_buffer_size": ""
            }
        },
        "resources": null
    },
    "io_config": {
        "attach_stderr": false,
        "attach_stdin": false,
        "attach_stdout": false,
        "open_stdin": false,
        "stdin_once": false,
        "tty": false
    },
    "config": {
        "env": [
           "CERT_FILE=/device.crt",
           "KEY_FILE=/device.key",
           "LOCAL_ADDRESS=tcp://mosquitto:1883",
           "LOG_FILE=",
           "LOG_LEVEL=INFO",
           "CA_CERT_PATH=/app/iothub.crt",
           "MESSAGE_MAPPER_CONFIG=/app/message-mapper-config.json",
           "ALLOWED_LOCAL_TOPICS_LIST=cloudConnector/#"
        ],
        "cmd": []
    },
    "network_settings": null,
    "state": {
        "pid": -1,
        "started_at": "",
        "error": "",
        "exit_code": 0,
        "finished_at": "",
        "exited": false,
        "dead": false,
        "restarting": false,
        "paused": false,
        "running": false,
        "status": "",
        "oom_killed": false
    },
    "created": "",
    "manually_stopped": false,
    "restart_count": 0
}